test.py

```diff
@@ -1,81 +1,53 @@
+import json
 import sys
 from time import time
 from time import sleep
-import argparse
-from argparse import ArgumentParser
-import json
 import numpy as np
 import cv2
+import struct
 
 import depthai
 
 import consts.resource_paths
-from depthai_helpers import utils
-
-def parse_args():
```

Comment:
See earlier comment about removing argument parsing (this should remain in the script)